src/test/java/resources/testData.json

```diff
@@ -0,0 +1,14 @@
+{
+  "testUrl": "https://www.flipkart.com/",
```

Comment:
lets make the key more specific like - `flipkartUrl` - since we would be having a lot of urls when we have more tests